src/Game/AI/Action/actionCurseRRematchCount.cpp

```diff
@@ -16,4 +18,32 @@ void CurseRRematchCount::loadParams_() {
     getDynamicParam(&mGameDataStringCounterName_d, "GameDataStringCounterName");
 }
 
+bool CurseRRematchCount::oneShot_() {
+    ChampionBalladManager* manager = ChampionBalladManager::instance();
+    if (!manager)
+        return false;
+
+    BlightType blight = BlightType(*mCurseRType_d);
```

Comment:
```suggestion
    auto blight = BlightType(*mCurseRType_d);
```
for the same reason as above. (also clang-tidy will complain if you don't use auto)